### PR TITLE
Setup Babel.js with Jasmine

### PR DESCRIPTION
**Pull request by piecioshka**
Hi @sebmck and @thejameskyle!

By this PR I wil try to explain, how to use Babel.js with Jasmine project.

Often I will setup unit tests by run command `jasmine init`.
This command creates simple json, eg:

```json
{
  "spec_dir": "spec",
  "spec_files": [
    "**/*[sS]pec.js"
  ],
  "helpers": [
    "helpers/**/*.js"
  ]
}
```

I would like to use ES6 syntax with my unit tests but without compiling it is imposible.
I love Babel.js and will try to help people how to (by adds simple line) write specs in ES6.

```json
{
  "helpers": [
    "../../node_modules/babel-core/register.js",
    "helpers/**/*.js"
  ]
}
```

After adding reference to `register.js` my specs (written in ES6) are properly parsed.

What are thinking about this?

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@
     mocha: Mocha
     jest: Jest
     karma: Karma
+    jasmine: Jasmine
```
Comment (thejameskyle) [inline _data/tools.yml]: mind alphabetizing these?

Comment (piecioshka) [inline _data/tools.yml]: Fixed. Sorry. My mistake.

Comment (thejameskyle) [inline _data/tools.yml]: No worries, it wasn't correct to begin with. Can you move mocha too?

Comment (piecioshka) [inline _data/tools.yml]: Done.

**Comment by thejameskyle** [comment]
Thanks

**Pull request merged**